tests/application/cms/test_views.py

```diff
@@ -416,42 +275,31 @@ def test_admin_user_can_see_publish_unpublish_buttons_on_edit_page(
     assert page.find_all("button", class_="button")[-1].text.strip().lower() == "unpublish"
 
 
-def test_internal_user_can_not_see_publish_unpublish_buttons_on_edit_page(
-    app, db, db_session, test_app_client, mock_rdu_user, stub_topic_page, stub_subtopic_page, stub_measure_page
-):
-
-    with test_app_client.session_transaction() as session:
-        session["user_id"] = mock_rdu_user.id
-
-    stub_measure_page.status = "DEPARTMENT_REVIEW"
-    db.session.add(stub_measure_page)
-    db.session.commit()
-
+def test_internal_user_can_not_see_publish_unpublish_buttons_on_edit_page(test_app_client, logged_in_rdu_user):
+    measure_version = MeasureVersionFactory(status="DEPARTMENT_REVIEW", published=False, published_at=None)
```

Comment:
`published` and `published_at` should be derived from the `DEPARTMENT_REVIEW` status - i.e. not need explicitly setting. If that doesn't work, the factory is probably not working as we want it to. 🤔

Reply:
I removed all the places I set `published` explicitly and tests still pass, so I guess it's working as expected :man_shrugging: